src/claims.rs

```diff
@@ -247,6 +247,15 @@ mod serde_timestamp {
                 .single()
                 .ok_or_else(|| E::custom("UTC timestamp overflow"))
         }
+
+        fn visit_f64<E>(self, value: f64) -> Result<Self::Value, E>
```

Comment:
Please add `#[allow(clippy::cast_possible_truncation)]` on this method and an explanation why truncation is OK in this case (`Utc.timestamp_opt()` will return no value in this case anyway).